lib/Raven/ReprSerializer.php

```diff
@@ -29,7 +29,11 @@ protected function serializeValue($value)
         } elseif (is_integer($value) || is_float($value)) {
             return (string) $value;
         } elseif (is_object($value) || gettype($value) == 'object') {
-            return 'Object '.get_class($value);
+            $repr = 'Object ' . get_class($value);
+            if (method_exists($value, 'toSentry')) {
+                $repr .= ' ' . $value->toSentry();
```

Comment:
I retained the class name by default because I guess is a valuable information I think it will go in virtually any `toSentry` method; what do you think guys? Perhaps we can remove the `Object` though.

About the fact of not locking the return type to be a string, I think that is a good suggestion, will fix it on next commit